api-docs/cloud-serverless/ref.yml

```diff
@@ -14306,14 +14353,31 @@ paths:
         description: Line protocol body
         required: true
       responses:
-        '204':
-          description: Write data is correctly formatted and accepted for writing to the bucket.
-        '400':
+        "204":
+          description: |
+            Success. All points in the batch are written and queryable.
+        "400":
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.